Makefile

```diff
@@ -18,14 +15,15 @@ help: ## This help.
 
 .PHONY: docker
 docker: ## Build Docker image
-	@(echo "Building docker image...")
-	docker build --file ./Dockerfile --build-arg VERSION=${VERSION} \
-		--build-arg BUILD_TIMESTAMP=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ") \
-		--build-arg COMMIT_HASH=$(shell git rev-parse HEAD) \
-		-t ${DOCKER_IMAGE}:${DOCKER_TAG} .
-
+	$(info "Building docker image...")
```

Comment:
why `info` instead of `echo`?

Reply:
I think we should use the `info/warning/error` Make utilities to display text, and use echo only if there's a variable to display.

links:
https://stackoverflow.com/a/29704625
https://www.gnu.org/software/make/manual/make.html#Make-Control-Functions

Although I have no problem reverting them back to echo if you like them better.

Reply:
I think this is a valid change, I'm not sure that it belongs in this PR. Adding driveby fixes like this disrupt the review process and also mess with the git history when it comes to identifying *why* a change was made, e.g. this doesn't have a commit message of its own describing the reasoning above. As more people start working on these projects async we need to be careful to keep the PR's in scope.

Reply:
Agree with @sambetts-cisco

Reply:
Alright, let me revert it then.